src/group_theory/subgroup.lean

```diff
@@ -147,7 +147,7 @@ instance : has_mem G (subgroup G) := ⟨λ m K, m ∈ (K : set G)⟩
 instance : has_coe_to_sort (subgroup G) := ⟨_, λ G, (G : Type*)⟩
 
 @[simp, norm_cast, to_additive]
-lemma mem_coe {K : subgroup G} [g : G] : g ∈ (K : set G) ↔ g ∈ K := iff.rfl
```

Comment:
I wonder why this one wasn't caught by the `incorrect_type_class_argument` linter. @robertylewis , are you surprised by this?

Reply:
A comment in that linter says:
> the head of the type should either unfold to a class, or be a local constant.
  A local constant is allowed, because that could be a class when applied to the
  proper arguments.